### ISSUE TITLE
Can you add a front-end example based on react or add a detailed guide on how to use it?

### ISSUE THREAD

**Issue by YonghoKwon** [EpicGames/PixelStreamingInfrastructure-archived-2024#236]
I would like to provide Pixel Streaming Service using @epicames-ps/lib-pixelstreamingfrontend-ue5.2, @epicames-ps/lib-pixelstreamingfrontend-ui-ue5.2 Library on previously developed screens (based on React).
"PixelStreamingInfrastructure/Frontend/implementations
/react"
It's not easy to apply with the source code in the above path, so could you add an example or a detailed guide on how to use it?

**Comment by lukehb** [comment]
Well https://github.com/EpicGames/PixelStreamingInfrastructure/tree/master/Frontend/implementations/react is only a very minimal example of how you might create a react app.

What specifically are you struggling with? If we know that we might be able to write some docs?

**Comment by vinchinzo123** [comment]
@lukehb in the provided React implementation there is not an example of how to use the UI Library `@epicgames-ps/lib-pixelstreamingfrontend-ui-ue5.X`. Can you please expand the React implementation to use the UI Library so developers can easily use the UI Feature buttons like "full screen", "info", and "settings"?

**Comment by lukehb** [comment]
@DenisTensorWorks Let's open an issue for the above request. Labels: good first issue and help wanted.

**Comment by lukehb** [comment]
@vinchinzo123 The the UI Library that we provide (e.g. @epicgames-ps/lib-pixelstreamingfrontend-ui) is an example of how you might build UI on top of the base library, but it itself is not the base for you to build your UI on top of if you are using a framework. If you are using a framework like react the idea is you will build the UI yourself using your framework and the base library. Said another way, our UI library is not designed so that you can add a framework to it, the UI library we provide is simply one example of how you might do UI using the base library.

In summary, maintaining multiple UI library implementations is outside the scope of this project at this stage, for now we will maintain only the typescript based UI library implementation.

**Comment by vinchinzo123** [comment]
@lukehb understood! Thanks for the clarification and good work on the Pixel Streaming Infrastructure, certainly not a simple task!
